app/static/css/style.css

```diff
@@ -1,64 +1,207 @@
+    --p-blue: #009FE3;
+    --p-grey: #4A4A4F;
+    --p-navy: #445261;
+    --p-red: #D63649;
+    --s-grey: #E6ECF0;
+    --s-yellow: #EFB920;
+    --s-pink: #FF5A60;
+    --s-teal: #23BBBB;
+    --t-grey: #E0E7FF;
+    --bg-grey: #F5F5F5;
+}
+
+/* ------------ Main Text Content ------------ */
+h1 {
+    font-family: 'Montserrat', sans-serif;
+}
+
+hr.divider {
+    width: 50px;
+    border-top: 3px solid var(--s-teal);
+    display: block;
+    margin: 0px auto;
+    background-color: var(--p-orange);
+    color: var(--p-orange);
+    height: auto;
+    opacity: 1;
+}
+
+/* ------------ Navigation ------------ */
+a,
 a.link-text {
     text-decoration: none;
-    color: rgb(41, 158, 158);
+    color: var(--s-teal);
+    font-weight: 500;
+    text-transform: uppercase;
 }
 
+a,
 a.link-text:hover {
-    color: rgb(16, 99, 99);
+    color: var(--p-navy);
 }
+/* ------------./end-navigation ------------ */
 
 
-/* Authentication Forms */
+/* ------------ Authentication Forms ------------ */
+h1.auth-header {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 600;
+    text-transform: uppercase;
+    font-size: 2rem;
+    text-shadow: 2px 2px 4px #00000047;
+}
 
 #registerForm {
-    align-items: center;
-    padding-top: 40px;
-    padding-bottom: 40px;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 400;
 }
 
-main.form-register,
-main.form-login {
-    display: block;
+main.container-fluid.p-3 {
+    /* background-color: var(--bg-grey); */
+    min-height: 100vh;
+    overflow: hidden;
+    margin: 0px auto;
+    background-color: var(--bg-grey);
+}
+
+
+input, input#username, input#password {
+    font-family: 'Montserrat', sans-serif;
+    color: var(--p-grey);
 }
 
 .card-wrapper {
     display: flex;
     justify-content: center;
-    min-width: 100%;
-    max-width: 800px;
+    margin: 0px auto;
+    /* padding: 15px; */
+}
+
+.card.card-dark {
+    -webkit-box-shadow: 5px 5px 20px var(--p-grey);
+    -moz-box-shadow: 5px 5px 20px  var(--p-grey);
+    box-shadow: 5px 5px 20px var(--p-navy);
+    border: none;
+    width: 400px;
     height: auto;
-    padding: 40px 40px;
-    margin: auto;
+    background-color: var(--p-navy);
+    color: var(--t-grey);
 }
 
-.card {
-    margin-top: 80px;
-    -webkit-box-shadow: 0 10px 6px -6px #777;
-    -moz-box-shadow: 0 10px 6px -6px #777;
-    box-shadow: 0 10px 6px -6px #777;
-    border: 1px solid rgb(194, 194, 194);
+.card.card-light {
+    -webkit-box-shadow: 5px 5px 20px var(--p-grey);
+    -moz-box-shadow: 5px 5px 20px  var(--p-grey);
+    box-shadow: 5px 5px 20px var(--p-navy);
+    border: none;
+    width: 400px;
+    height: auto;
+    background-color: var(--bg-grey);
+    color: var(--p-grey);
 }
 
-form.login-form,
-form.register-form {
-    margin: 30px 80px;
-    display: block;
+.card-header.card-header-dark {
+    background-color: var(--p-navy);
+    color: var(--t-grey);
+    border: none;
+    margin: 0px auto;
+    padding: 0px 0px;
```

Comment:
`padding: 0px 0px;` ? 🤔 Isn't this redundant? Can we remove this property and value all together? Or, alternatively, if it needs to stay, consider using shorthand: `padding: 0;`